README.md

```diff
@@ -2,11 +2,13 @@
 
 # Hello microverse
 
-> One paragraph statement about the project.
+> The fierst project in microverse.
 
 ![screenshot](./app_screenshot.png)
 
-Additional description about the project and its features.
+## Features
+
+- Home page
```

Comment:
- [ ] You should update your README file. You can use this [template](https://github.com/microverseinc/readme-template) and make sure there are no  broken links(example screenshot) and empty sections. If you won't use some sections delete them. 
- [ ] **I recommend use most of section as you can**